src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameType7.java

```diff
@@ -0,0 +1,17 @@
+/*
+AbbreviationAsWordInName
+allowedAbbreviations = ORDER, OBSERVATION, UNDERSCORE, TEST
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+public class InputAbbreviationAsWordInNameType7 {
+    void getORDER_OBSERVATION() {} // ok
+
+    void getNONE_Test() {} // violation
```

Comment:
Please restore all test code and add more for violation cases and make even more to weird cases.

Input should be full of test code.
User examples in xdoc should be compact to show concept of validation.

Reply:
Ok ?